packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -11,22 +11,22 @@ export const sliderClassNames: SlotClassNames<SliderSlots> = {
   input: 'fui-Slider__input',
 };
 
-// Internal CSS variables. Private until we want to commit to them
-const thumbSizeVar = `--fui-slider-thumb-size`;
-const railSizeVar = `--fui-slider-rail-size`;
-const railColorVar = `--fui-slider-rail-color`;
-const progressColorVar = `--fui-slider-progress-color`;
-const thumbColorVar = `--fui-slider-thumb-color`;
+// Internal CSS variables
+const thumbSizeVar = `--fui-Slider__thumb--size`;
```

Comment:
nit: You could reference the `sliderClassNames` object here so it's always consistent:

```suggestion
const thumbSizeVar = `--${sliderClassNames.thumb}--size`;
```

Reply:
I think the goal here was to keep the strings intact to make finding them easier.